src/main/java/net/sf/jabref/logic/importer/ImportFormatPreferences.java

```diff
@@ -0,0 +1,81 @@
+package net.sf.jabref.logic.importer;
+
+import java.nio.charset.Charset;
+import java.util.Set;
+
+import net.sf.jabref.logic.bibtex.FieldContentParserPreferences;
+import net.sf.jabref.logic.importer.fileformat.CustomImporter;
+import net.sf.jabref.logic.labelpattern.LabelPatternPreferences;
+import net.sf.jabref.preferences.JabRefPreferences;
+
+public class ImportFormatPreferences {
+
+    private final Set<CustomImporter> customImportList;
+
+    private Charset encoding;
```

Comment:
I'm fine with this (and the "setter" below) but @simonharrer usually immutable preferences classes.